### PR TITLE
add rimas bio

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

File: bios/rimantas-mocevicius.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Rimantas Mocevicius
+
+![rimas](../images/headshots/rimantas-mocevicius.jpeg)
+
+[<img src="../images/icons/twitter.png" width="56px;"/>](https://twitter.com/Rimusz) [<img src="../images/icons/github.png" width="56px;"/>](https://github.com/rimusz) [<img src="../images/icons/linkedin.png" width="56px;"/>](https://linkedin.com/in/rimantasmocevicius)
+
+Rimas is big Open Source supporter. He co-founded Helm and wrote CoreOS Essentials book.
+He is currently the Senior Developer at Jfrog.
```
Comment (retgits) [inline bios/rimantas-mocevicius.md]: Please change Jfrog to JFrog (uppercase `F`)

File: bios/rimantas-mocevicius.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Rimantas Mocevicius
+
+![rimas](../images/headshots/rimantas-mocevicius.jpeg)
+
+[<img src="../images/icons/twitter.png" width="56px;"/>](https://twitter.com/Rimusz) [<img src="../images/icons/github.png" width="56px;"/>](https://github.com/rimusz) [<img src="../images/icons/linkedin.png" width="56px;"/>](https://linkedin.com/in/rimantasmocevicius)
+
+Rimas is big Open Source supporter. He co-founded Helm and wrote CoreOS Essentials book.
```
Comment (retgits) [inline bios/rimantas-mocevicius.md]: I'd suggest rephrasing this a bit to: "_Rimas is a big supporter of Open Source software. He co-founded [Helm](https://helm.sh/) and wrote the CoreOS Essentials book._"

**Review by retgits — COMMENTED** [review]
I've added two comments to update the text slightly

File: bios/rimantas-mocevicius.md (hunk 2 above)
Comment (rimusz) [reply bios/rimantas-mocevicius.md]: done

**Pull request merged**